src/main.f90

```diff
@@ -179,6 +182,10 @@ program quick
       call gpu_upload_xyz(xyz)
       call gpu_upload_atom_and_chg(quick_molspec%iattype, quick_molspec%chg)
     endif
+
+!#ifdef CEW
+    call gpu_set_cew(.true.,ierr) 

```

Comment:
`quick_cew_module` contains an `upload()` subroutine, which accepts an error flag. Although that error flag does not do anything because the implementation of `gpu_set_cew` in `gpu.cu` does not have it.

Reply:
Why do we need this here and not only in the API module?

Reply:
Fixed.